.github/ISSUE_TEMPLATE/question.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Question
+about: A question about the spec or processes 
+title: ""
+labels: '❔ Question'
+assignees: ''
+---
+
+<!--
+
+!!! IMPORTANT !!!
+
+Before creating your issue:
+
+* **Have a question?** Find community resources at [asyncapi/community](https://www.github.com/asyncapi/community).
```

Comment:
you have some plans to extend https://www.github.com/asyncapi/community with some resources? maybe better link here https://github.com/asyncapi/community/issues/1 ?

Reply:
That's a good point. I'll change it.

Reply:
Changed!